ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -224,13 +249,46 @@ module.exports = Marionette.LayoutView.extend({
     },
     saveTemplateToBackend: function() {
         let loadingView = new LoadingView();
+        let _this = this;
+        let _user = user;
         $.ajax({
             url: '/search/catalog/internal/forms/query',
             data: JSON.stringify(this.getQueryAsQueryTemplate()),
             method: 'PUT',
             contentType: 'application/json',
             customErrorHandling: true
         })
+        .done((data, textStatus, jqxhr) => {
+            announcement.announce({
+                title: 'Saved!',
+                message: 'Search form has been saved.',
+                type: 'success'
+            });
+
+            let queryTemplate = _this.getQueryAsQueryTemplate();
+            let sorts = queryTemplate['querySettings'] && queryTemplate['querySettings'].sorts;
+
+            if (sorts) {
+                sorts = sorts.map(sort => ({ attribute: sort.split(',')[0], direction: sort.split(',')[1] }));
+
+            queryTemplate = {
+                type: 'custom',
+                name: queryTemplate['title'],
+                filterTemplate: JSON.stringify(queryTemplate['filterTemplate']),
+                src: (queryTemplate['querySettings'] && queryTemplate['querySettings'].src) || '',
+                federation: (queryTemplate['querySettings'] && queryTemplate['querySettings'].federation) || 'enterprise',
+                querySettings: sorts, 
```

Comment:
❗️ There are a lot of other things that can go into `querySettings` than just sorts (in fact, `src` and such are typically a part of that)

See [QuerySettings.js](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/js/model/QuerySettings.js)